app/configurations/config.hbnext.js

```diff
@@ -185,16 +185,15 @@ export default configMerger(walttiConfig, {
                     de: 'https://stuttgart.stadtmobil.de/privatkunden/',
                     en: 'https://stuttgart.stadtmobil.de/privatkunden/',
                 },
```

Comment:
Could you please add some code so this can be activated with an env variable?

Reply:
Already working on that

Reply:
> Could you please add some code so this can be activated with an env variable?

@leonardehrenfried Done. Just add CARGO_BIKES_ENABLED=true

Reply:
Just found a little bug here. The station is still clickable although the icon is not there. Working on that.

Reply:
Done. Now it works.